src/workers.ts

```diff
@@ -23,7 +23,7 @@ import { settings } from './settings';
  * The default worker file locations
  */
 const DEFAULT_WORKERS = {
-  penumbra: '/worker.penumbra.js',
+  penumbra: 'worker.penumbra.js',
```

Comment:
@eligrey is this change ok? I've seen it go back and forth in this PR. With the `/` the worker 404s in the example